README.md

```diff
@@ -6,16 +6,21 @@ Provide common management and troubleshooting tools for the Rook Ceph storage pr
 
 ## Install
 
-To install tool without krew
+To install tool
 
-1. Download the [kubectl-rook-ceph.sh](kubectl-rook-ceph.sh)
-2. ```sudo install kubectl-rook-ceph.sh /usr/local/bin/kubectl-rook_ceph```
+1. Download the release zip file
+
+    ```curl -OJL https://github.com/rook/kubectl-rook-ceph/archive/v0.0.1.zip```
+
+2. Run the command below to use it with krew (this step is temporary until we push our manifest to krew-index)
+
+    ```$ kubectl krew install --manifest=rook-ceph.yaml --archive=<PATH_TO_ZIP_FILE>```
```

Comment:
After we merge our manifest to Krew, I thought the install would just be:
```
kubectl krew install rook-ceph
```
And there should be no need to download the separate archive, right? We might use the `--archive` option for development, but we should focus this readme on end-user usage.

Reply:
right. I'll remove it once it is available with krew

Reply:
yes, I'll update soon as it is available in krew